.claude/skills/implement/SKILL.md

```
@@ -50,9 +50,12 @@ Use Task tool to delegate. Pass arguments via prompt text:
 - `test-validator`: Test rules + Spec coverage
     - Args: `files` (required), `spec_directory` (optional)
 
-- `code-style-validator`: ktlint validation
+- `code-style-validator`: code style validator
     - Args: none (runs on entire project)
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)